compiler/src/iree/compiler/Codegen/Common/EncodingUtils.h

```diff
@@ -89,15 +89,31 @@ bool isMatmulEncodingUser(IREE::LinalgExt::EncodingUser user);
 /// Returns if encoding user is one of batch matmul encodings.
 bool isBatchMatmulEncodingUser(IREE::LinalgExt::EncodingUser user);
 
+bool getIntOrZero(IntegerAttr a);
+
+/// Returns if the encoding is a vecmat.
```

Comment:
It is missing `true` in the sentence? Same for below comments.

```suggestion
/// Returns true if the encoding is a vecmat.
```